sci-misc/boinc/boinc-7.16.16.ebuild

```diff
@@ -86,16 +95,28 @@ pkg_setup() {
 }
 
 src_prepare() {
+	# bug #732024
+	if test "x$(get_libdir)" = "xlib64"; then
+	    sed -i -e 's,/:/lib:/usr/lib:,:/lib64:/usr/lib64:,g' m4/sah_check_lib.m4 || die
+	fi
+
 	default
 
 	# prevent bad changes in compile flags, bug 286701
 	sed -i -e "s:BOINC_SET_COMPILE_FLAGS::" configure.ac || die "sed failed"
 
 	eautoreconf
+
+	use X && setup-wxwidgets
+
+	# bug #732024
+	if test "x$(get_libdir)" = "xlib64"; then
+	    sed -i -e 's,/lib\([ /;:"]\),/lib64\1,g' configure || die
+	fi
 }
 
 src_configure() {
-	use X && setup-wxwidgets unicode
+	LDFLAGS="-L${EPREFIX}/usr/$(get_libdir) -L${EPREFIX}/$(get_libdir) ${LDFLAGS}" \
```

Comment:
have you tried `append-libs`? Generally injecting linker dirs into `LDFLAGS` is bad style, and seeing this package uses autotools, `LIBS` is generally respected

Reply:
It was important to prepend the directories, not to append them, to fix some issues. But maybe not any more? I'll try this out, give a me a minute...

Reply:
append-libs works just fine. Thank you very much!